util/topgen/lib.py

```diff
@@ -70,6 +70,25 @@ def get_module_by_name(top, name):
     return module
 
 
+def get_package_name_by_intermodule_signal(top, struct) -> str:
+    """Search inter-module signal package with the struct name
+
+    For instance, if `flash_ctrl` has inter-module signal package,
+    this function returns the package name
+    """
+    instances = top["module"] + top["memory"]
+
+    intermodule_instances = [
+        x["inter_signal_list"] for x in top["module"] + top["memory"]
+        if "inter_signal_list" in x
+    ]
+
+    for m in intermodule_instance:
```

Comment:
@eunchan sorry i'm a bit confused, is this section complete?

because i don't actually see intermodule_instance defined anywhere (i do see intermodule_instance**S**)

Reply:
haha. Nope. this function is not used YET. I planned to use this function to define the struct signals in the top .sv but not yet. If I called this function in the top, I would see the MISTAKE :) Good catch!